index.html

```diff
@@ -65,67 +65,10 @@ <h3>Multi-Post Stories</h3>
-                <div class="works">
-                    <h2>Etch-A-Sketch</h2>
-                    <p>A recreation of the Etch-A-Sketch interactive dashboard where you can select the quantity of spaces and the colors.</p>
-                    <ul class="lw languages">
-                        <li>HTML</li>
-                        <li>CSS</li>
-                        <li>Javascript</li>
-                    </ul>
-                    <a href="https://luffytaro22.github.io/Etch-A-Sketch/" target="_blank" rel="noopener" class="hide"><button type="button">See project</button></a>
-                </div>
-                <div class="works">
-                    <h2>Rock-Paper-Scissors</h2>
-                    <p>The global-popular game against the computer. Will you be winner or the losser? Try it out!</p>
-                    <ul class="lw languages">
-                        <li>HTML</li>
-                        <li>CSS</li>
-                        <li>Javascript</li>
-                    </ul>
-                    <a href="https://luffytaro22.github.io/Rock-Paper-Scissors/" target="_blank" rel="noopener" class="hide"><button type="button">See project</button></a>
-                </div>
-                <div class="works">
-                    <h2>Calculator</h2>
-                    <p>A simple calculator that can do the basics operations.</p>
-                    <ul class="lw languages">
-                        <li>HTML</li>
-                        <li>CSS</li>
-                        <li>Javascript</li>
-                    </ul>
-                    <a href="https://luffytaro22.github.io/Calculator/" target="_blank" rel="noopener" class="hide"><button type="button">See project</button></a>
-                </div>
-                <div class="works">
-                    <h2>Calculator</h2>
-                    <p>A simple calculator that can do the basics operations.</p>
-                    <ul class="lw languages">
-                        <li>HTML</li>
-                        <li>CSS</li>
-                        <li>Javascript</li>
-                    </ul>
-                    <a href="https://luffytaro22.github.io/Calculator/" target="_blank" rel="noopener" class="hide"><button type="button">See project</button></a>
-                </div>
-                <div class="works">
-                    <h2>Calculator</h2>
-                    <p>A simple calculator that can do the basics operations.</p>
-                    <ul class="lw languages">
-                        <li>HTML</li>
-                        <li>CSS</li>
-                        <li>Javascript</li>
-                    </ul>
-                    <a href="https://luffytaro22.github.io/Calculator/" target="_blank" rel="noopener" class="hide"><button type="button">See project</button></a>
-                </div>
             </div>
+            <div id="pop-up">
+            </div>
+            <script src="projects.js"></script> <!-- The projects cards -->
```

Comment:
- [ ] Following the best practices, You don't add Javascript files at the center of your code, You can add this at the bottom of this file or at the top using the defer keyword. :+1: :100: